templates/org/aac/aac-design-pattern.md

```diff
@@ -60,16 +56,14 @@ Evaluate how to use <!--insert name of pattern--> in a workload's design to addr
 | [Operational Excellence](/azure/well-architected/operational-excellence/checklist) helps deliver **workload quality** through **standardized processes** and team cohesion. | <!--Explanation--><br><br> - <!--[Guide code and relevant guide name](<link>)--><br> - <!--[Guide code and relevant guide name](<link>)--> |
 | [Performance Efficiency](/azure/well-architected/performance-efficiency/checklist) helps your workload **efficiently meet demands** through optimizations in scaling, data, and code. | <!--Explanation--><br><br> - <!--[Guide code and relevant guide name](<link>)--><br> - <!--[Guide code and relevant guide name](<link>)--> |
 
-Consider any trade-offs against the goals of the other pillars that this pattern might introduce.
+If this pattern introduces trade-offs within a pillar, consider them against the goals of the other pillars.
```

Comment:
@ckittel - Does this phrasing preserve the intended meaning? Just trying to make this sentence easier to parse for a machine reader or a non-native English speaker.